ote/src/cljs/ote/app/state.cljs

```diff
@@ -29,3 +29,17 @@
 (defonce viewer (r/atom {:loading? true
                          :url nil
                          :geojson nil}))
+
+
+(defn windowresize-handler
+  "When window size changes set width and height to app state"
+  [event]
+  (let [w (.-innerWidth js/window)
+        h (.-innerHeight js/window)]
+    (.log js/console "resize")
+    (do
```

Comment:
unnecessary `do` and both width and height can be `assoc`ed with the same call.
`(swap! app assoc :width w :height h)`

Reply:
Fixed